# 1. Variables
- A variable is a container that stores data

### Declaration (Creating a Variable)
Format: `variable_name = value`

In [1]:
name = "Alice"
age = 25
price = 19.99

### Using (Calling) a Variable
- Use `print()` to display the value
- Or just write the variable name (acts as print in Jupyter)

In [2]:
name = "Bob"

print(name)    # Using print()
name           # Just the variable name

Bob


'Bob'

### Assigning New Values
- You can change the value anytime

In [3]:
score = 10
print(score)

score = 20      # Assign new value
print(score)

score = score + 5
print(score)

10
20
25


# 2. Constants
- A value that should NOT change
- Written in ALL CAPITAL LETTERS

### Declaration (Creating a Constant)
Format: `CONSTANT_NAME = value`

In [4]:
PI = 3.14159
MAX_SPEED = 120
TAX_RATE = 0.15

### Using (Calling) a Constant
- Same as variables

In [5]:
PI = 3.14159

print(PI)      # Using print()
PI             # Just the constant name

3.14159


3.14159

### Using Constants in Calculations

In [6]:
PI = 3.14159
radius = 5

area = PI * radius ** 2
print("Area:", area)

Area: 78.53975


# 3. Comparison
- **Variables**: lowercase names, can change → `age = 25`
- **Constants**: ALL_CAPS names, should not change → `PI = 3.14`
- **Calling**: Use `print()` or just the name in Jupyter